src/core/DSPCoreValueSets.ttl

```diff
@@ -286,8 +291,12 @@ DSPCoreValueSets:PlateletType
   rdfs:subClassOf DSPCoreValueSets:Blood ;
 .
 DSPCoreValueSets:PrimaryCell
-  a owl:Class ;
+  a DSPCoreValueSets:PrimaryCellType ;
   rdfs:label "Primary cell" ;
+.
+DSPCoreValueSets:PrimaryCellType
+  a owl:Class ;
+  rdfs:label "PrimaryCell" ;
```

Comment:
Just asking since I'm unfamiliar with this - I would have expected to see "rdfs:label "PrimaryCell" under the "DSPCoreValueSets:PrimaryCell" and vice versa. Is that not the case here?

Reply:
I am reusing the label which is a bit unusual. But we're only using this to categorize data and you can't refer to the category unless there's an instance.  So I think it's safe; the ids are different; it's just that the human-readable label is reused.  You cannot use a category (class) where you use an object (an instance).

Reply:
Ah okay, that makes sense to me.